keyboards/keycapsss/kimiko/kimiko.c

```diff
@@ -0,0 +1,246 @@
+        oled_write_P(on_on_1, false);
+    } else if(modifiers & MOD_MASK_CTRL) {
+        oled_write_P(on_off_1, false);
+    } else if(modifiers & MOD_MASK_SHIFT) {
+        oled_write_P(off_on_1, false);
+    } else {
+        oled_write_P(off_off_1, false);
+    }
+
+    if(modifiers & MOD_MASK_SHIFT) {
+        oled_write_P(shift_on_1, false);
+    } else {
+        oled_write_P(shift_off_1, false);
+    }
+
+    if(modifiers & MOD_MASK_CTRL) {
+        oled_write_P(ctrl_on_2, false);
+    } else {
+        oled_write_P(ctrl_off_2, false);
+    }
+
+    if (modifiers & MOD_MASK_CTRL & MOD_MASK_SHIFT) {
+        oled_write_P(on_on_2, false);
+    } else if(modifiers & MOD_MASK_CTRL) {
+        oled_write_P(on_off_2, false);
+    } else if(modifiers & MOD_MASK_SHIFT) {
+        oled_write_P(off_on_2, false);
+    } else {
+        oled_write_P(off_off_2, false);
+    }
+
+    if(modifiers & MOD_MASK_SHIFT) {
+        oled_write_P(shift_on_2, false);
+    } else {
+        oled_write_P(shift_off_2, false);
+    }
+}
+
+
+void render_logo(void) {
+    static const char PROGMEM kimiko_logo[] = {
+        0xA0, 0xA1, 0xA2, 0xA3, 0xA4,
+        0xC0, 0xC1, 0xC2, 0xC3, 0xC4, 0};
+    oled_write_P(kimiko_logo, false);
+    // oled_write_P(PSTR("Kimiko"), false);
+}
+
+void render_layer_state(void) {
+	static const char PROGMEM default_layer[] = {
+		0x20, 0x94, 0x95, 0x96, 0x20,
+		0x20, 0xb4, 0xb5, 0xb6, 0x20,
+		0x20, 0xd4, 0xd5, 0xd6, 0x20, 0};
+	static const char PROGMEM raise_layer[] = {
+		0x20, 0x97, 0x98, 0x99, 0x20,
+		0x20, 0xb7, 0xb8, 0xb9, 0x20,
+		0x20, 0xd7, 0xd8, 0xd9, 0x20, 0};
+	static const char PROGMEM lower_layer[] = {
+		0x20, 0x9a, 0x9b, 0x9c, 0x20,
+		0x20, 0xba, 0xbb, 0xbc, 0x20,
+		0x20, 0xda, 0xdb, 0xdc, 0x20, 0};
+	static const char PROGMEM adjust_layer[] = {
+		0x20, 0x9d, 0x9e, 0x9f, 0x20,
+		0x20, 0xbd, 0xbe, 0xbf, 0x20,
+		0x20, 0xdd, 0xde, 0xdf, 0x20, 0};
+
+	if(layer_state_is(_ADJUST)) {
+		oled_write_P(adjust_layer, false);
+	} else if(layer_state_is(_LOWER)) {
+		oled_write_P(lower_layer, false);
+	} else if(layer_state_is(_RAISE)) {
+		oled_write_P(raise_layer, false);
+	} else {
+		oled_write_P(default_layer, false);
+	}
+	#ifdef OLED_DISPLAY_128X64
+		oled_write_P(PSTR("\n"), false);
+	#endif
+}
+
+void render_status_main(void) {
+    render_space();
+    render_space();
+    render_logo();
+    render_space();
+    render_layer_state();
+    render_space();
+    render_mod_status_gui_alt(get_mods()|get_oneshot_mods());
+    render_mod_status_ctrl_shift(get_mods()|get_oneshot_mods());
+}
+
+void render_status_secondary(void) {
+    render_space();
+    render_space();
+    render_logo();
+    render_space();
+}
+
+bool oled_task_kb(void) {
+    if (!oled_task_user()) {
+        return false;
+    }
+
+    if (is_keyboard_master()) {
+        render_status_main();  // Renders the current keyboard state (layer, lock, caps, scroll, etc)
+    } else {
+        render_status_secondary();
+    }
+
+    return true;
```

Comment:
`true` or `false`

Reply:
IIRC, it doesn't matter here, other than the function requires it to be set.  True should be fine.